src/css/profile/mobile/theme-changeable/theme.color.less

```diff
@@ -265,7 +265,6 @@
 //						DropdownMenu
 //***************************************************************************
 @color_dropdownmenu_list_group_stroke: B0733L1; // #[color] list stroke line color
-@color_dropdownmenu_list_item_bg_press: B042; // #[color] list item background press color
```

Comment:
I think I did mistake in comment.
You can remove this line and replace this variable in:
![image](https://user-images.githubusercontent.com/29534410/77621893-96170280-6f3d-11ea-94cf-981cf283ecfc.png)
by `var(--ripple-color)`

Reply:
But I don't sure if the press effect is on dropdown menu items.

Reply:
Yes, @lslachciak noticed that ripple-effect is missing. However, proposed change is not sufficient to have ripple-effect working - it does not disappear. There is a separate issue https://github.com/Samsung/TAU/issues/832 for this and let's take care of this separately.